FreewheelSampleApp/app/src/main/java/com/ooyala/sample/players/PreconfiguredFreewheelPlayerActivity.java

```diff
@@ -53,6 +60,36 @@ public final static String getName() {
   @Override
   public void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    // Here, thisActivity is the current activity
+    if (ContextCompat.checkSelfPermission(this,
+            Manifest.permission.WRITE_EXTERNAL_STORAGE)
+            != PackageManager.PERMISSION_GRANTED) {
+
+      // Should we show an explanation?
+      if (ActivityCompat.shouldShowRequestPermissionRationale(this,
```

Comment:
ok. Can you please fix it as per the standard. Feel free to modify it. our main objective to give permission to the sample app for write log in logFile which get created in external storage while running the automation.